katello/katello-backup

```diff
@@ -10,11 +10,14 @@ require 'highline/import'
 
 EXCLUDED="--exclude goferd,foreman-proxy,squid,smart_proxy_dynflow_core,qdrouterd,qpidd"
 DATABASES = ['pulp', 'pgsql', 'mongodb']
+FOREMAN_DB_CONFIG = '/etc/foreman/database.yml'
+CANDLEPIN_DB_CONFIG = '/etc/candlepin/candlepin.conf'
```

Comment:
Instead of reading these 2 files, I think it would be a lot simpler to read `/etc/foreman-installer/scenarios.d/#{scenario}-answers.yaml` ?
You could get the scenario from `/etc/foreman-installer/scenarios.d/last_scenario.yaml`